.vscode/launch.json

```diff
@@ -19,9 +19,7 @@
       "program": "${workspaceFolder}/pkg/cmd/grafana/",
       "env": {},
       "cwd": "${workspaceFolder}",
-      "args": ["apiserver", 
-          "--secure-port=8443", 
-          "--runtime-config=testdata.datasource.grafana.app/v0alpha1=true"]
+      "args": ["apiserver", "--secure-port=8443", "--runtime-config=testdata.datasource.grafana.app/v0alpha1=true"]
```

Comment:
Did some prettier tidy up here as well (my editor was wanting to change it anyway). I can revert it if people would rather